packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -18,24 +25,60 @@ class AndroidCameraCameraX extends CameraPlatform {
     CameraPlatform.instance = AndroidCameraCameraX();
   }
 
-  /// ProcessCameraProvider used to get list of cameras. Visible only for testing.
+  // Instances used to access camera functionality:
+
+  /// The [ProcessCameraProvider] instance used to access camera functionality.
   @visibleForTesting
   ProcessCameraProvider? processCameraProvider;
 
-  /// Camera selector used to determine which CameraInfos are back cameras.
+  /// The [Camera] instance returned by the [processCameraProvider] when a [UseCase] is
+  /// bound to the lifecycle of the camera it manages.
+  @visibleForTesting
+  Camera? camera;
+
+  // Instances used to configure and bind use cases to ProcessCameraProvider instance:
+
+  /// The [Preview] instance that can be configured to present a live camera preview.
   @visibleForTesting
-  CameraSelector? backCameraSelector = CameraSelector.getDefaultBackCamera();
+  Preview? preview;
 
-  /// Camera selector used to determine which CameraInfos are back cameras.
+  /// Whether or not the [preview] is currently bound to the lifecycle that the
+  /// [processCameraProvider] tracks.
   @visibleForTesting
-  CameraSelector? frontCameraSelector = CameraSelector.getDefaultFrontCamera();
+  bool previewIsBound = false;
+
+  bool _previewIsPaused = false;
+
+  // Instances used for camera configuration:
+
+  /// The [CameraSelector] used to configure the [processCameraProvider] to use
+  /// the desired camera.
+  @visibleForTesting
+  CameraSelector? cameraSelector;
+
+  /// The controller we need to broadcast the different camera events.
+  ///
+  /// It is a `broadcast` because multiple controllers will connect to
+  /// different stream views of this Controller.
+  /// This is only exposed for test purposes. It shouldn't be used by clients of
+  /// the plugin as it may break or change at any time.
+  @visibleForTesting
+  final StreamController<CameraEvent> cameraEventStreamController =
+      StreamController<CameraEvent>.broadcast();
+
+  /// The stream of camera events.
+  Stream<CameraEvent> _cameraEvents(int cameraId) =>
+      cameraEventStreamController.stream
+          .where((CameraEvent event) => event.cameraId == cameraId);
+
+  // Implementation of Flutter camera platform interface (CameraPlatform):
```

Comment:
I don't think this line is helpful since this class extends only `CameraPlatform` anyways.